### PR TITLE
Migrate content from MediaWiki bot page on wiki to user manual

### LINKED ISSUE #3575: Migrate content from MediWiki bot page on wiki to user manual
New user manual has details about creating a bot for Wikidata https://docs.openrefine.org/manual/wikidata#manage-wikidata-account

There is a page on the wiki that gives more general information about creating a bot account for any Wikibase instance https://github.com/OpenRefine/OpenRefine/wiki/MediaWiki-Bot-Passwords

The more general information should be migrated across to the new user manual and then the MediaWiki-Bot-Passwords page in the wiki can be deprecated (replace the content with a link to the relevant page in the new user manual)

### PR DESCRIPTION
**Pull request by khadijm**
Fixes #3575

### CODE REVIEW

**Comment by khadijm** [comment]
preview 
Screenshot from 2022-03-28 22-14-32

Screenshot from 2022-03-28 22-14-49

please @wetneb  and @ostephens  can you review my PR  thanks

File: docs/docs/manual/wikibase/overview.md [hunk 1]
```diff
@@ -96,6 +96,24 @@ If your account or your bot is not properly authorized, OpenRefine will not disp
 
 You can store your unencrypted username and password in OpenRefine, saved locally to your computer and available for future use. For security reasons, you may wish to leave this box unchecked. You can also save your OpenRefine-specific bot password in your browser or with a password management tool. 
 
+Official manual of bot passwords: https://www.mediawiki.org/wiki/Manual:Bot_passwords
```
Comment (wetneb) [inline docs/docs/manual/wikibase/overview.md:99]: I would rather avoid adding a URL in plain text like this (rather create a hyperlink with a meaningful label).

File: docs/docs/manual/wikibase/overview.md [hunk 2]
```diff
@@ -96,6 +96,24 @@ If your account or your bot is not properly authorized, OpenRefine will not disp
...
+
+> Bot passwords allow applications or bots to log in with a simple username-password combination even if the wiki employs some extra authorization steps that the bot framework could not handle; they also allow limiting the permissions given to the bot (e.g. the bot might be allowed to edit articles but not to create new ones). Clients using bot passwords can only access the API, not the normal web interface.
+
+For a given Wikibase instance, you can create bot passwords at `https://${Wikibase site}/wiki/Special:BotPasswords`.
+
+We take Wikidata for example here:
+
+1. Goto https://www.mediawiki.org/wiki/Special:BotPasswords.
```
Comment (wetneb) [inline docs/docs/manual/wikibase/overview.md:107]: `Goto` should be `Go to`

**Review by wetneb — COMMENTED** [review]
Thank you! The content you added seems largely redundant with the lines before (lines 88 to 97). Perhaps it would be better to rework those and make them applicable to other Wikibase instances than Wikidata?

**Comment by khadijm** [comment]
new preview 
Screenshot from 2022-04-02 09-34-41

**Review by wetneb — COMMENTED** [review]
I still see a lot of redundancy between your additions and the contents of the previous section, for instance lines 89 to 91 duplicate your line 113.

**Comment by khadijm** [comment]
please @wetneb  can you check again if there is still redundancy. thanks

**Comment by wetneb** [comment]
I am not so sure about this change: the part that you removed also covers normal passwords that are not mentioned in your "bot password" section.

**Comment by khadijm** [comment]
yep

**Review by wetneb — CHANGES_REQUESTED** [review]
There is still the same redundancy of content I mentioned earlier.

**Comment by wetneb** [comment]
I would like to close this PR soon because it does not look like it is getting in a mergeable state.

**Pull request closed without merging**